internal/gatewayapi/testdata/httproute-with-header-filter-duplicate-removes.out.yaml

```diff
@@ -62,10 +62,6 @@ httpRoutes:
       conditions:
       - type: Accepted
         status: "True"
```

Comment:
ditto

Reply:
Commit e14991a updates the PR to only set the "UnsupportedValue" reason when `Accepted=false`.